soh/soh/Enhancements/debugger/SohConsoleWindow.cpp

```diff
@@ -14,7 +14,7 @@ void SohConsoleWindow::UpdateElement() {
 void SohConsoleWindow::DrawElement() {
     UIWidgets::PushStyleInput(THEME_COLOR);
     // Small font (13) to match hardcoded width values in the LUS window.. set large font after below TODO addressed
-    ImGui::PushFont(OTRGlobals::Instance->fontMonoSmall);
+    ImGui::PushFont(OTRGlobals::Instance->fontMonoLarger);
```

Comment:
I had comments here explaining the issue with the larger fonts. The "Submit" button gets clipped away losing letters.

I think small font would be preferable over the button being clipped.